.github/workflows/sync-issue-labels-add.yml

```
@@ -0,0 +1,111 @@
+      cancel-in-progress: true
+    runs-on: hl-sdk-py-lin-md
+    steps:
+      - name: Harden the runner
+        uses: step-security/harden-runner@a90bcbc6539c36a85cdfeb73f7e2f433735f215b # v2.15.0
+        with:
+          egress-policy: audit
+
+      - name: Download labels artifact
+        id: download
+        continue-on-error: true
+        uses: actions/download-artifact@70fc10c6e5e1ce46ad2ea6f2b72d43f7d47b13c3 # v8.0.0
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)